communities/plymouth-university/community-assessment.md

```diff
@@ -0,0 +1,122 @@
+# Plymouth University
+
+Plymouth University has been having troubles with the tech community for a number of years. We've seen a seady decline in the number of tech based societies and events since around 2012. We are now in a situation where the last community left is the Game Dev Society (GDSoc). Thankfully there are still a few pockets of students who are enthusiastic about various strands of tech.
```

Comment:
When I entered to college we were on a similar situation. The student groups were declining rapidly until the point where only one was active (suspiciously, also a game dev community) but their events were on a small scale. Fortunate enough we had a bunch of students attending hackathons and meetups and we joined to revive the official student group of the school of engineering. We were succesfull during our administration and group is currently on the 3rd generation.